builder/ncloud/step_get_rootpassword.go

```diff
@@ -0,0 +1,55 @@
+package ncloud
+
+import (
+	ncloud "github.com/NaverCloudPlatform/ncloud-sdk-go/sdk"
+	"github.com/hashicorp/packer/packer"
+	"github.com/mitchellh/multistep"
+)
+
+type StepGetRootPassword struct {
+	Conn            *ncloud.Conn
+	GetRootPassword func(serverInstanceNo string, privateKey string) (string, error)
+	Say             func(message string)
+	Error           func(e error)
+}
+
+func NewStepGetRootPassword(conn *ncloud.Conn, ui packer.Ui) *StepGetRootPassword {
+	var step = &StepGetRootPassword{
+		Conn:  conn,
+		Say:   func(message string) { ui.Say(message) },
+		Error: func(e error) { ui.Error(e.Error()) },
+	}
+
+	step.GetRootPassword = step.getRootPassword
+
+	return step
+}
+
+func (s *StepGetRootPassword) getRootPassword(serverInstanceNo string, privateKey string) (string, error) {
+	reqParams := new(ncloud.RequestGetRootPassword)
+	reqParams.ServerInstanceNo = serverInstanceNo
+	reqParams.PrivateKey = privateKey
+
+	rootPassword, err := s.Conn.GetRootPassword(reqParams)
+	if err != nil {
+		return "", err
+	}
+
+	return rootPassword.RootPassword, nil
+}
+
+func (s *StepGetRootPassword) Run(state multistep.StateBag) multistep.StepAction {
+	s.Say("Get Root Password")
+
+	serverInstanceNo := state.Get("InstanceNo").(string)
+	loginKey := state.Get("LoginKey").(*LoginKey)
+
+	rootPassword, err := s.GetRootPassword(serverInstanceNo, loginKey.PrivateKey)
```

Comment:
If running with debug this password should be logged.

Reply:
Added logging for root password